doc/tutorial/basic/tutorial.rst

```diff
@@ -234,21 +234,21 @@ persistence model, `pickle <https://docs.python.org/2/library/pickle.html>`_::
   0
 
 In the specific case of scikit-learn, it may be more interesting to use
-joblib's replacement for pickle (``joblib.dump`` & ``joblib.load``),
+joblib's replacement for pickle in utils (``dump`` & ``load``),
 which is more efficient on big data but it can only pickle to the disk
 and not to a string::
 
-  >>> from sklearn.externals import joblib
-  >>> joblib.dump(clf, 'filename.joblib') # doctest: +SKIP
+  >>> from joblib import dump, load
+  >>> dump(clf, 'filename.joblib') # doctest: +SKIP
 
 Later, you can reload the pickled model (possibly in another Python process)
 with::
 
-  >>> clf = joblib.load('filename.joblib') # doctest:+SKIP
+  >>> clf = load('filename.joblib') # doctest:+SKIP
 
 .. note::
 
-    ``joblib.dump`` and ``joblib.load`` functions also accept file-like object
+    ``dump`` and ``load`` functions also accept file-like object
```

Comment:
I prefer explicit, `joblib.dump` instead of `dump` here and in the docstring above, so it's not confused with `pickle.dump` any other library that implements the `dump/load` methods.